cli/cmd/lql_sources.go

```diff
@@ -0,0 +1,78 @@
+//
+// Author:: Salim Afiune Maya (<afiune@lacework.net>)
+// Copyright:: Copyright 2020, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+const (
+	lqlSourcesDebugMsg    string = "retrieving LQL data sources"
+	lqlSourcesNotFoundMsg string = "There were no data sources found.\n"
+	lqlSourcesUnableMsg   string = "unable to retrieve LQL data sources"
+)
+
+var (
+	// lqlSourcesCmd represents the lql data sources command
+	lqlSourcesCmd = &cobra.Command{
+		Use:   "sources",
```

Comment:
I vote to name this command `list-sources` and have an alias for `sources`
```suggestion
		Use:     "list-sources",
		Aliases: []string{"sources"},
```